### PR TITLE
Conformance Test Results for MetaHub v2

### PR DESCRIPTION
**Pull request by ChristianKniep**
Update to MetaHub with all test completing...

Signed-off-by: Christian Kniep <christian@qnib.org>

### CODE REVIEW

File: distribution-spec/v1.0/metahub/README.md [hunk 1]
```diff
@@ -3,3 +3,33 @@
...
+a2f2473b6a7e: Pull complete
+Digest: sha256:fa4073c84baeda1fa38e12f7d0675476b435ebb5e70048ff7bc728aa3ab2dd9b
+Status: Downloaded newer image for metahub-registry.org/qnib/featuretest:latest
+>> This container is optimized for: amd64
+```
+
+Next we use the profile `zen3` to pull down a new image optimized for the particular hardware architecutre.
+
+```bash
+$ echo hub |docker login metahub-registry.org --password-stdin  -u meta
```
Comment (sudo-bmitch) [inline distribution-spec/v1.0/metahub/README.md:26]: Should the user here be `meta/zen3`?

Comment (ChristianKniep) [reply distribution-spec/v1.0/metahub/README.md:26]: the user carries the information on which specific platform the client logs in from. 

- `meta[/generic]` -> generic
- `meta/zen3` -> zen3

Comment (sudo-bmitch) [reply distribution-spec/v1.0/metahub/README.md:26]: That was my impression looking at the website. But it looks like the second example shows using `meta` user to pull the `zen3` image.

**Review by sudo-bmitch — APPROVED** [review]
LGTM

**Comment by sudo-bmitch** [comment]
Looks like the DCO was missed on the second commit, we need that to merge: https://github.com/opencontainers/oci-conformance/pull/96/checks?check_run_id=9826401306

**Comment by ChristianKniep** [comment]
I missed the sign, one sec

**Comment by sudo-bmitch** [comment]
I'm just noticing now that the reproducibility steps doesn't have the various variables and commands run. They would be helpful. Here's an example: https://conformance.opencontainers.org/static/v1.0/instructions/bundle-bar/

**Comment by ChristianKniep** [comment]
Hey, @sudo-bmitch. I rewrote some of the backend, so it took more time - but I finally got around to it. I also added some word how to get a user.

**Review by jdolitsky — CHANGES_REQUESTED** [review]
to add to what brandon was saying, i think we are looking for what values to set for the OCI_ env vars, for example:

```
export OCI_ROOT_URL="https://bundle.bar"
export OCI_USERNAME="${BB_USER}"
export OCI_PASSWORD="${BB_TOKEN}"
export OCI_NAMESPACE="u/${BB_USER}/conformance/primary"
export OCI_CROSSMOUNT_NAMESPACE="u/${BB_USER}/conformance/secondary"
export OCI_TEST_PULL=1
export OCI_TEST_PUSH=1
export OCI_TEST_CONTENT_DISCOVERY=1
export OCI_TEST_CONTENT_MANAGEMENT=1
```

also, looks like DCO still fails

**Comment by ChristianKniep** [comment]
arg, always miss the signoff - sorry for that. I amended it.

**Review by jdolitsky — APPROVED** [review]

**Pull request merged**